### DISCUSSION TITLE
Changing the context path defaults?

### DISCUSSION THREAD

**Discussion by zubcevic** [WebGoat/WebGoat#1669] [Ideas]
After the last changes the context paths are now fully configurable. Should we change the defaults for both of them to / or just keep it like it is? And only change it when you want a specific lab experience?

**Comment by aolle** [comment]
If I understood you correctly, I like how it is, the /WebGoat is fine. However, I have always thought that it would be great if the request to / automatically redirected to /WebGoat. But such a change might potentially affect some lessons (or not); it would need thorough testing.
